core/app/[locale]/not-found.tsx

```diff
@@ -41,8 +42,8 @@ async function getFeaturedProducts(): Promise<CarouselProduct[]> {
   return productCardTransformer(featuredProducts, format);
 }
 
-export default async function NotFound() {
-  const t = await getTranslations('NotFound');
+export default function NotFound() {
+  const t = useTranslations('NotFound');
```

Comment:
Doesn't this still have to be async as this page is not a client component?

Reply:
Yeah, I think you're right.

Reply:
I mean, it works fine this way, `useTranslations` can be used in shared components, but I will revert.